package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@isaac.frontend/testcafe-reporter-allure",
-  "version": "0.1.2",
+  "version": "0.0.0-development",
```

Comment:
Doesn't this have to be v0.1.2?

Reply:
No, by default the actual version is only added to the package.json that is published (in npm) by Semantic Release. The version in the source-code defaults to 0.0.0-development.

https://github.com/semantic-release/semantic-release/blob/master/docs/support/FAQ.md#why-is-the-packagejsons-version-not-updated-in-my-repository